### DISCUSSION TITLE
SNS 챌린지에 대해 질문드립니다

### DISCUSSION THREAD

**Discussion by fazerlover** [devrel-kr/HackaLearn#158] [1 Q&A - HackaLearn 운영] [answered]
안녕하세요. 
SNS 챌린지에 대해 질문드립니다.
처음에 신청할 때 썼던 계정과 다른 계정에 올려도 되는지 문의드리고 싶습니다.
감사합니다:)

**Comment by justinyoo** [comment, marked as answer]
@fazerlover 괜찮습니다! 저희에게 알려주셔요. hackalearn.korea@outlook.com 으로 관련 내용 이메일 보내주세요.

**Reply by fazerlover** [reply]
감사합니다!!!
